src/modules/menstrual-period/menstrual-period.service.ts

```diff
@@ -95,4 +95,21 @@ export class MenstrualPeriodService {
   toLocalDate(date: Date) {
     return new Date(date.getTime() + date.getTimezoneOffset() * 60000);
   }
+
+  async deleteDate(id: number) {
+    let dateRepository = await this.menstrualPeriodDateRepository.findOneBy({
+      id,
+    });
+    if (dateRepository) {
+      await this.menstrualPeriodDateRepository.delete({ id });
+      return {
+        code: 'success',
+      };
+    } else {
+      throw new CustomConflictException({
+        code: 'date-id-not-not-exists',
+        message: 'This date not exists',
```

Comment:
Aqui só corrigir a digitação: 
date-id-does-not-exist
This date does not exist